src/libexpr/eval.hh

```diff
@@ -65,8 +65,12 @@ std::string printValue(const EvalState & state, const Value & v);
 std::ostream & operator << (std::ostream & os, const ValueType t);
 
 
-// FIXME: maybe change this to an std::variant<SourcePath, URL>.
-typedef std::pair<std::string, std::string> SearchPathElem;
+struct SearchPathElem
+{
+    std::string prefix;
+    // FIXME: maybe change this to an std::variant<SourcePath, URL>.
+    std::string path;
```

Comment:
What do these fields contain? How are they used? A little bit of doc comment goes a long way here.